extensions/amp-story/0.1/amp-story.js

```diff
@@ -609,15 +609,19 @@ export class AmpStory extends AMP.BaseElement {
     const platform = Services.platformFor(this.win);
     if (platform.isSafari() || platform.isIos()) {
       this.mutateElement(() => {
-        setStyle(this.element, 'display', 'none');
+        setImportantStyles(this.element, {
```

Comment:
I thought we only want to set initial amp-story styles as important.

Reply:
ah sorry, my bad. Didn't see that the one in file is also not important

Reply:
Done